src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -312,6 +322,15 @@ static void window_guest_list_mouseup(rct_window *w, rct_widgetindex widgetIndex
         window_invalidate(w);
         w->scrolls[0].v_top = 0;
         break;
+    case WIDX_FILTER_BY_NAME:
+        if (strnlen(_window_guest_list_filter_name, sizeof(_window_guest_list_filter_name)) > 0) {
```

Comment:
This does not adhere to the code style. It can be found on the wiki. One of the things is that braces ( { and } ) go onto their own lines.